packages/forma-36-react-components/src/components/Table/Table.tsx

```diff
@@ -1,36 +1,33 @@
-import React, { Component } from 'react';
+import React, { HTMLProps } from 'react';
 import cn from 'classnames';
 import styles from './Table.css';
 
-export interface TableProps {
+export interface TableProps extends HTMLProps<HTMLTableElement> {
   testId?: string;
   className?: string;
-  style?: React.CSSProperties;
   children: React.ReactNode;
 }
 
 const defaultProps: Partial<TableProps> = {
   testId: 'cf-ui-table',
 };
 
-export class Table extends Component<TableProps> {
-  static defaultProps = defaultProps;
+export const Table = (props: TableProps) => {
+  const { className, children, testId, ...otherProps } = props;
```

Comment:
Wdyt about putting the default props here during this refactor?

Reply:
tbh, I don't know the difference, all form controls refactor uses `defaultProps` I'm happy to change if we want it always to be defined inline

Reply:
One reason to make them inline is that we won't need to do `Partial<TableProps>` for the `defaultProps` anymore

More info here https://github.com/contentful/forma-36/pull/647#discussion_r526932324

Reply:
It's not urgent tough!